.chloggen/fp-deprecate-old-configs.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: filterprocessor
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Deprecation warning for all non-OTTL configuration options. Use OTTL configuration options instead. All non-OTTL configuration options will be removed in 0.74.0.
```

Comment:
I agree with @Aneurysm9 that v0.74.0 is too aggressive, should we just say 'in a future version' in this PR? This component is in alpha, but it's quite old so it's probably widely used.